test/Feature/QuerySpec.hs

```diff
@@ -295,13 +284,8 @@ spec actualPgVersion = do
         [json|[{"id":1,"commenter_id":1,"user_id":2,"task_id":6,"content":"Needs to be delivered ASAP","users_tasks":{"taskId": 6}}]|]
         { matchHeaders = [matchContentTypeJson] }
 
-    it "embed data with two fk pointing to the same table" $
-      get "/orders?id=eq.1&select=id, name, billing_address_id(id), shipping_address_id(id)" `shouldRespondWith`
-        [str|[{"id":1,"name":"order 1","billing_address_id":{"id":1},"shipping_address_id":{"id":2}}]|]
```

Comment:
Same here. Moved to [EmbedDisambiguationSpec](https://github.com/steve-chavez/postgrest/blob/5e7d7ea8efa8ea32641c82ed5b6ca11472a27a78/test/Feature/EmbedDisambiguationSpec.hs#L243-L246). The QuerySpec tests were getting big and unrelated.